packages/cli-lib/lib/config.js

```diff
@@ -602,6 +630,10 @@ const deleteEmptyConfigFile = () => {
   );
 };
 
+const deleteConfigFile = () => {
```

Comment:
Should `deleteEmptyConfigFile` call this new delete function? Just to DRY things up a little

Reply:
`deleteConfigFile` is more destructive in that it will delete a config file that is not empty, however, `deleteEmptyConfigFile` will only delete an empty config file.